app/controllers/AmountsController.scala

```diff
@@ -1,13 +1,13 @@
 package controllers
 
 import com.gu.googleauth.AuthAction
-import models.AmountsRegions
+import models.ConfiguredAmounts
+import models.ConfiguredAmounts._
 import play.api.mvc.{AnyContent, ControllerComponents}
-import io.circe.generic.auto._
```

Comment:
it turns out that asking circe to do its magic here is a bit unreliable, so we now define the Encoder/Decoder as implicit vals and import them here